.github/workflows/release.yml

```diff
@@ -0,0 +1,395 @@
+name: Release
+
+on:
+  push:
+    tags:
+    - 'v*'
+
+jobs:
+  create-release:
+    runs-on: ubuntu-latest
+    outputs:
+      upload_url: ${{ steps.create_release.outputs.upload_url }}
+      app_version: ${{ steps.define_version.outputs.app_version }}
+    steps:
+      - name: Define version
+        id: define_version
+        run: |
+          APP_VERSION=${GITHUB_REF/refs\/tags\/v/}
+          echo "Tag: $GITHUB_REF Version = $APP_VERSION"
+          echo "::set-output name=app_version::$APP_VERSION"          
+      - name: Create Release
+        id: create_release
+        uses: actions/create-release@v1
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        with:
+          tag_name: ${{ github.ref }}
+          release_name: Release ${{ github.ref }}
+          draft: true
+          prerelease: false
+
+  linux-jpackage:
```

Comment:
I see that now we duplicate the jobs from build.yml, but adding notarization and signing. But I don't know if that can be avoided...

Reply:
There are probably too many subtle differences between creating the jpackage installers and native images for a release versus a regular build. But you could merge both workfiles and for instance only run the steps that are related to attaching the artifacts to a github release, by adding an `if` condition:

```
      - name: Upload Release Asset (deb)
        id: upload-release-asset-deb
        if: ${{ github.event_name == 'push' }}
        uses: actions/upload-release-asset@v1
        ...
```